common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -142,7 +142,8 @@
             log: function(eventName, data) {
                 var logInfo = _.extend({
                     id: this.state.id,
-                    code: this.state.isYoutubeType() ? this.state.youtubeId() : 'html5'
+                    // eslint-disable-next-line no-nested-ternary
+                    code: this.state.isYoutubeType() ? this.state.youtubeId() : this.state.canPlayHLS ? 'hls' : 'html5'
```

Comment:
Note we will have to modify the data pipeline to support this change as well, see code like this:

https://github.com/edx/edx-analytics-pipeline/blob/4c05666880d71fdbba6f5ea0e88a42fa089b8303/edx/analytics/tasks/insights/video.py#L214

We will also likely want to warn the research community